rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -1,5 +1,5 @@
 Name:		securedrop-workstation-dom0-config
-Version:	0.11.1rc2
+Version:	0.11.1
```

Comment:
Do you want to update the date in the RPM changelog too?

Reply:
ooops. yup!